src/core/test_helpers/kbn_server.ts

```diff
@@ -6,31 +6,22 @@
  * Side Public License, v 1.
  */
 
-import type { KibanaClient } from '@elastic/elasticsearch/api/kibana';
 import { ToolingLog, REPO_ROOT } from '@kbn/dev-utils';
 import {
-  // @ts-expect-error https://github.com/elastic/kibana/issues/95679
```

Comment:
not sure it's a correct issue, I don't see any type failures due to type collision between `mocha` and `jest`